api/src/pcapi/core/finance/factories.py

```diff
@@ -264,13 +264,22 @@ class Meta:
     venue = factory.SubFactory(offerers_factories.VenueFactory)
     kind = models.IncidentType.OVERPAYMENT
     status = models.IncidentStatus.CREATED
+    details = {
+        "origin": "Demande e-mail",
+        "createdAt": datetime.datetime.utcnow(),
+        "author": "Sandy Box",
+        "validator": "",
+        "validatedAt": "",
+    }
 
 
 class IndividualBookingFinanceIncidentFactory(BaseFactory):
     class Meta:
         model = models.BookingFinanceIncident
 
-    booking = factory.SubFactory(bookings_factories.BookingFactory)
+    booking = factory.SubFactory(
+        bookings_factories.ReimbursedBookingFactory,
+    )
```

Comment:
Je commente rarement sur ce genre de reformatage, mais ici c'est inutile et ça nuit à la lisibilité.

```suggestion
    booking = factory.SubFactory(bookings_factories.ReimbursedBookingFactory)
```

Reply:
C'est à cause de la virgule, je suis d'accord.